pkgs/development/libraries/sope/default.nix

```diff
@@ -16,6 +16,13 @@
     ++ optionals (mysql != null) [ libmysqlclient mysql ]
     ++ optional (postgresql != null) postgresql);
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/inverse-inc/sope/pull/50.diff";
```

Comment:
https://github.com/inverse-inc/sope/commit/45c1819d71bd8c7d14719a687cde5e12ed8800d9.patch

Reply:
pull requests can change, commits don't.

Reply:
Also, it would be useful  to add a comment here so people see what the patch does instead of having to open it.

Reply:
and that it will be included in the next release of sogo.

Reply:
Oh, last time I checked they actually weren't merged yet.  Besides that you're of course right anyways. ;)